src/platform/macos.rs

```diff
@@ -70,6 +76,16 @@ impl WindowExtMacOS for Window {
     fn set_has_shadow(&self, has_shadow: bool) {
         self.window.set_has_shadow(has_shadow)
     }
+
+    #[inline]
+    fn get_window_level(&self) -> i64 {
+        self.window.get_window_level()
+    }
+
+    #[inline]
+    fn set_window_level(&self, level: i64) {
+        self.window.set_window_level(level)
+    }
```

Comment:
This should instead use the `set_level_async` in `util/async.rs` to make this call thread-safe.